docs/metrics-storage-management/reference-streaming.mdx

```diff
@@ -0,0 +1,481 @@
+| `default port`                                  | `19999`                   | The port to use if `destination` does not specify one.                                                                                                                                                                                                                                                                                                    |
+| [`send charts matching`](#send-charts-matching) | `*`                       | A space-separated list of [Netdata simple patterns](/libnetdata/simple_pattern/README.md) to filter which charts are streamed. [Read more &rarr;](#send-charts-matching)                                                                                                                                                                                  |
+| `buffer size bytes`                             | `1048576`                 | The size of the buffer to use when sending metrics. The default `1048576` equals a buffer of 1MB, which is good for 10-20 seconds of data. Increase this if you expect latencies higher than that. The buffer is flushed on reconnect.                                                                                                                    |
+| `reconnect delay seconds`                       | `5`                       | How long to wait until retrying to connect to the parent node.                                                                                                                                                                                                                                                                                            |
+| `initial clock resync iterations`               | `60`                      | Sync the clock of charts for how many seconds when starting.                                                                                                                                                                                                                                                                                              |
+| **`[API_KEY]` section**                         |                           | For configuring a parent node using an `API_KEY`.                                                                                                                                                                                                                                                                                                         |
```

Comment:
I will give a suggestion, instead to have all parameters in an unique table, I think it will be more clear for users, if we treat the sections in different tables.

Reply:
Great suggestion—let me know what you think about the revision I just commited!

Reply:
LGTM! :) 
I will approve your PR after this commit. Thank you @joelhans !